wp-content/plugins/core/src/Object_Meta/Theme_Options.php

```diff
@@ -0,0 +1,100 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Project\Object_Meta;
+
+use Tribe\Libs\ACF;
+
+class Theme_Options extends ACF\ACF_Meta_Group {
+
+	public const NAME = 'theme_options';
+
+	 // Analytics
+	public const ANALYTICS_TAB    = 'tab_analytics';
+	public const ANALYTICS_GTM_ID = 'analytics_gtm_id';
+
+	 // Social Media
+	public const SOCIAL_TAB       = 'tab_social';
+	public const SOCIAL_FACEBOOK  = 'social_facebook';
+	public const SOCIAL_TWITTER   = 'social_twitter';
+	public const SOCIAL_YOUTUBE   = 'social_youtube';
+	public const SOCIAL_LINKEDIN  = 'social_linkedin';
+	public const SOCIAL_PINTEREST = 'social_pinterest';
+	public const SOCIAL_INSTAGRAM = 'social_instagram';
+
+	public function get_keys(): array {
+		return [
+		   static::ANALYTICS_GTM_ID,
+		   static::SOCIAL_FACEBOOK,
+		   static::SOCIAL_TWITTER,
+		   static::SOCIAL_YOUTUBE,
+		   static::SOCIAL_LINKEDIN,
+		   static::SOCIAL_PINTEREST,
+		   static::SOCIAL_INSTAGRAM,
+		];
+	}
+
+	/**
+	 * @param string|int $key
+	 * @param string|int $post_id
+	 *
+	 * @return mixed|null
+	 */
+	public function get_value( $key, $post_id = 'option' ) {
+		return in_array( $key, $this->get_keys(), true ) ? get_field( $key, $post_id ) : null;
+	}
+
+	public function get_group_config(): array {
+		$group = new ACF\Group( self::NAME, $this->object_types );
+		$group->set( 'title', __( 'Theme Options', 'tribe' ) );
```

Comment:
```suggestion
		$group->set( 'title', esc_html__( 'Theme Options', 'tribe' ) );
```

The remaining labels in here also need escaping.